pkg/registry/etcd/ns_server.go

```diff
@@ -131,7 +131,7 @@ func (n *etcdNSRegistryServer) sendEvent(resp *registry.NetworkServiceResponse)
 
 func (n *etcdNSRegistryServer) Register(ctx context.Context, request *registry.NetworkService) (*registry.NetworkService, error) {
 	meta := metav1.ObjectMeta{
-		GenerateName: "nse-",
+		GenerateName: "netsvc-",
```

Comment:
Can this change cause compatibility issues during upgrade?

Reply:
I believe it's can't produce compatibility issues, it's related to case when applied network service via NSM api without a name.